rlp/main.cpp

```diff
@@ -189,18 +155,18 @@ void putOut(bytes _out, Encoding _encoding, bool _encrypt, bool _quiet)
 
 	switch (_encoding)
 	{
-	case Encoding::Hex: case Encoding::Auto:
-		cout << toHex(_out) << endl;
-		break;
-	case Encoding::Base64:
-		cout << toBase64(&_out) << endl;
-		break;
-	case Encoding::Binary:
-		cout.write((char const*)_out.data(), _out.size());
-		break;
-	case Encoding::Keccak:
-		cout << sha3(_out).hex() << endl;
-		break;
+		case Encoding::Hex: case Encoding::Auto:
```

Comment:
Don't change the indentation of `cases`s inside switches, I just fixed it before in my commit here